drivers/display/display_ili9342c.c

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Unless there is a special reason (for instance SETEXTC going first), it would be good to follow the register address order.